src/library/scala/collection/IterableOnce.scala

```diff
@@ -1138,7 +1147,7 @@ trait IterableOnceOps[+A, +CC[_], +C] extends Any { this: IterableOnce[A] =>
     *  @param sep   the separator string.
     *  @return      the string builder `b` to which elements were appended.
     */
-  def addString(b: StringBuilder, sep: String): StringBuilder = addString(b, "", sep, "")
+  @inline final def addString(b: StringBuilder, sep: String): StringBuilder = addString(b, "", sep, "")
```

Comment:
Since you're optimizing, has anyone asked if it's worth adding the other implementations? Arguably `xs.mkString(", ")` is the common case. 1 vs 3 appends? Or maybe strings are assumed slow and one only cares about space. I think I only `src.mkString` when slurping as in `BufferedSource`.

Reply:
Yes, I'm raising the question of whether passing the empty strings is highly optimized, or if it makes no difference worth mentioning. I don't know the answer.

Reply:
not sure, I think empty strings are OK since they're consts